.github/workflows/release-build.yaml

```diff
@@ -0,0 +1,47 @@
+name: Docker Build Publish
+
+on:
+  release:
+    types: [published]
```

Comment:
Is there a reason to use `release published` vs `tag pushed`? If you use tag pushed, you can combine the build-test and this workflow into one to keep things in one workflow (and less duplicated code), like https://github.com/kiva/kiva/blob/development/.github/workflows/docker-build-push.yaml.

Also, the way things are setup currently, both `build-push-test` and this workflow will both run when a new release is created because this one runs on `release published` and the other on `tag pushed`, which is something the `create-release` flow does both of.

Reply:
Whoops - thanks for calling that out! I had separated them for clarity, but it sounds like that made it more confusing. Let me look at recombining these 👀

Reply:
Ah, my bad @hobbsh - I got distracted by other things and completely forgot to circle back to this 🤦🏻‍♀️